pkg/cmd/workspace/code.go

```diff
@@ -196,5 +196,5 @@ func openIDE(ideId string, activeProfile config.Profile, workspaceId string, pro
 var ideFlag string
 
 func init() {
-	CodeCmd.Flags().StringVarP(&ideFlag, "ide", "i", "", "Specify the IDE ('vscode' or 'browser')")
+	CodeCmd.Flags().StringVarP(&ideFlag, "ide", "i", "", "Specify the IDE ('vscode', 'browser', 'jetbrains', 'cursor', 'ssh')")
```

Comment:
The goal of the issue is not just to hard-code the IDE IDs.
It is more to expand the functionality here, to create a function that returns all the implemented IDEs IDs and to construct a list of them for the output here so we can be assured that, if we implement support for new IDEs in the future, we won't have to update the hard-coded list here manually.

You can also think of introducing maybe constants for the IDE IDs so the code is reusable and not error-prone to IDE ID label changing.

Reply:
Sure! I will make suggested changes 😄

Reply:
@lbrecic Done! Have a look